### PR TITLE
Fix inconsistent benchmarking throughput/time

### LINKED ISSUE #185: Error while running benchmark.py on GPU
Running benchmark.py on Nvidia A100 SXM4  produces an error:

Exception: Error occurred while computing benchmark on device File "tools/benchmarking/benchmark.py", line 177, in distribute
    raise Exception(f"Error occurred while computing benchmark on device {job}") from exception
Exception: Error occurred while computing benchmark on device <Future at 0x7f05c18ac310 state=finished raised Exception>

Note: In my setup this error does not occur when running on CPU only. It does not occur either when running on CPU and GPU for patchcore model. It occurs when running on GPU for padim or cflow

### PR DESCRIPTION
**Pull request by ashwinvaidya17**
# Description

- Supress outputs from model and print values from benchmarking script

- Fixes #185 

## Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the [pre-commit style and check guidelines](https://openvinotoolkit.github.io/anomalib/guides/using_pre_commit.html#pre-commit-hooks) of this project.
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes

### CODE REVIEW

File: tools/benchmarking/benchmark.py [hunk 1]
```diff
@@ -218,6 +256,8 @@ def sweep(run_config: Union[DictConfig, ListConfig], device: int = 0, seed: int
 
     # Run benchmarking for current config
     model_metrics = get_single_model_metrics(model_config=model_config, openvino_metrics=convert_openvino)
+    print(model_config.model.name, model_config.dataset.category)
```
Comment (samet-akcay) [inline tools/benchmarking/benchmark.py:259]: What if we want to run `benchmark.py` for custom datasets that don't have `category` field?

File: tools/benchmarking/benchmark.py [hunk 2]
```diff
@@ -41,9 +45,42 @@
     set_in_nested_config,
 )
 
+warnings.filterwarnings("ignore")
+
 logger = logging.getLogger(__file__)
+logging.getLogger("pytorch_lightning").setLevel(logging.ERROR)
+logging.getLogger("torchmetrics").setLevel(logging.ERROR)
+logging.getLogger("os").setLevel(logging.ERROR)
```
Comment (samet-akcay) [inline tools/benchmarking/benchmark.py:53]: maybe we could set the level in a for loop to avoid the duplicated lines

**Review by samet-akcay — CHANGES_REQUESTED** [review]
Thanks, just minor stuff

**Review by djdameln — COMMENTED** [review]
This works, but there will still be inconsistency in the throughput numbers between a normal training run and the benchmarking script. Maybe to avoid confusion, you could also add the batch size to the output in the timer callback Something like `Throughput (batch_size=16): 12 FPS`

**Review by djdameln — APPROVED** [review]

**Pull request merged**